File: Task/Sum-of-first-n-cubes/Zen-C/sum-of-first-n-cubes.c
import "locale.h"

fn main() {
    setlocale(LC_NUMERIC, "en_US.UTF-8");
    println "Sums of the first 'n' cubes (0 <= n < 50):";
    let sum = 0;
    for i in 0..50 {
        sum += i * i * i;
        print "{sum:'9d} ";
        if i % 10 == 9 { println ""; }
    }
}

File: Task/Sum-of-first-n-cubes/Go/sum-of-first-n-cubes.go
package main

import (
    "fmt"
    "rcu"
)

func main() {
    fmt.Println("Cumulative sums of the first 50 cubes:")
    sum := 0
    for n := 0; n < 50; n++ {
        sum += n * n * n
        fmt.Printf("%9s ", rcu.Commatize(sum))
        if n%10 == 9 {
            fmt.Println()
        }
    }
    fmt.Println()
